packaged/shared.sh

```diff
@@ -57,3 +57,16 @@ function get_lisk_custom_config() {
 	fi
 	echo "$LISK_CUSTOM_CONFIG"
 }
+
+function get_config() {
+# use first of: custom configuration file, network configuration file or default configuration file
+	local KEY=$1
+	VALUE=$( jq --raw-output "$KEY" "$LISK_CUSTOM_CONFIG" )
```

Comment:
Shouldn't we set `LISK_CUSTOM_CONFIG` here?

Reply:
It's admittedly slightly evil. It wouldn't fly in strict mode.

LISK_CUSTOM_CONFIG gets defined in both lisk.sh and lisk_snapshot.sh. If it's not found, get_config will try the config based on LISK_NETWORK and if that fails, default to config/default/config.json

Reply:
How about `LISK_CUSTOM_CONFIG=${LISK_CUSTOM_CONFIG:-$1}`?

Reply:
Moved default value of LISK_CUSTOM_CONFIG into packaged/shared.sh